templates/base/packages/nextjs/scaffold.config.ts.template.mjs

```diff
@@ -9,9 +9,11 @@ const defaultScaffoldConfig = {
     onlyLocalBurnerWallet: true,
   };
 
-const contents = ({ preConfigContent, configOverrides, extraConfigTypeName }) => {
+const contents = ({ preConfigContent, configOverrides, extraConfigTypeName, skipLocalChainInTargetNetworks }) => {
   // add solidityFramework network
-  const targetNetworks = configOverrides.map(override => override.targetNetworks).flat();
+  let targetNetworks = configOverrides.map(override => override.targetNetworks).flat();
+  // if skipLocalChainInTargetNetworks is true, don't include solidityFramework network
+  targetNetworks = skipLocalChainInTargetNetworks ? targetNetworks.slice(1) : targetNetworks;
```

Comment:
will slice one network if solidityFramework in `none`

Reply:
> will slice one network if solidityFramework is none

didn't get this, like we won't slice the targetNetworks right because `skipLocalChainInTargetNetworks ` is false when you select `solidityFramework ` as none? 

I just tried creating an instance with none options and it nicely loaded `targetNetworks: [chains.mainnet],`

Reply:
Oh I didnt take into account that `skipLocalChainInTargetNetworks` doesn't make sense when `solidityFramework` is none. 

In any case, when I'm running command from initial message
```
yarn build:dev && yarn cli -e https://github.com/scaffold-eth/create-eth-extensions/tree/extra-beta-examples
```

with `-s hardhat` I'm getting 
```
targetNetworks: [customChain, chains.sepolia],
```
but it should be with `hardhat`, right?

And if running with `-s none` I'm getting 
```
targetNetworks: [chains.sepolia],
```
without `customChain`.


Or it's expected and I'm missing something?

Reply:
Just fixed it https://github.com/scaffold-eth/create-eth/pull/253/commits/18cdb8c22077c4917e4c885c380be0cd6620fd68, since the args are arrays had check for 0th index for truth or falsy value